capstone/src/main/java/com/google/sps/data/FeatureData.java

```diff
@@ -0,0 +1,129 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+/* This class stores data on various features in the model */
+public class FeatureData {
+  private static ArrayList<Precinct> precinctData;
+
+  private final int NUMBER_OF_PRECINCTS = 10;
+
+  /* The precinct data was taken from public datasets such as
```

Comment:
If possible, please follow the rules here with with block comment style:
https://google.github.io/styleguide/javaguide.html#s4.8.6-comments will help with readability in the future!